app.rb

```diff
@@ -22,9 +22,15 @@ class Iam < Sinatra::Base
   # to make sure every record is collected into the hierarchy
   default_client = Client.find_or_create(name: 'default',
                                          description: 'The default client')
-  Project.find_or_create(name: 'default',
-                         client_id: default_client.id,
-                         description: 'The default project')
+  default_project = Project.find_or_create(name: 'default',
+                                           client_id: default_client.id,
+                                           description: 'The default project')
+  default_sku = Sku.find_or_create(name: 'default',
+                                   description: 'The default SKU')
+  NodeResourcesProject.find_or_create(project_id: default_project.id,
```

Comment:
I don' know if this is necessary - first, these don't associate an actual node or db, so I am not sure what function they serve, and since we don't have a default node or default db, I don't know that we need any default associations created at all, just the project and client objects.